src/styles/main.scss

```diff
@@ -1,14 +1,11 @@
-html {
-  font-family: Roboto, sans-serif;
-}
+@import "./catalog";
```

Comment:
Import file paths should be updated to have an underscore prefix and .scss extension, like './_catalog.scss' and './_header.scss'